### ISSUE TITLE
Bug: Fx Text entry getting "stuck" (no EL recognition)

### ISSUE THREAD

**Issue by Lykdraft** [kmatheussen/radium#521]
So there was really something to it.

When entering FX Text values, the EL (Edit Line) Setting sometimes doesnt work.

I dont know exactly how to recreate this, it *might* have something to do with enabled sublines, that you can't see, not sure. 

Take a look at the .gif, you will see I have an EL Setting of 2, the cursor is set correct (you can see the value and also the automation line changing) but the EL will not move.

Best,
tL.

http://makeagif.com/8mnNmc

**Comment by kmatheussen** [comment]
It's a little bit hard to see what happens in the video, so when you say "EL setting sometimes doesn't work", do you mean that the cursor only moves one step, or that it doesn't move at all?

If it doesn't move at all, it might have something to do with the play cursor. There might be some exceptions that tells the program not to jump down when playing, but when using the play cursor, the cursor should of course jump down whether we are playing or not.

**Comment by Lykdraft** [comment]
Yeah, doesn't move at all.  So entry itself works, but it just doesnt move, no matter which EL Value I use.

Strange was when I loaded the .gif animation software I had Radium in the background minimized.
When I came back to Radium iit worked again for some brief period of time. Then it ceased working again.

**Comment by kmatheussen** [comment]
Okay, it's probably got something to do with changing line then, and not something with the EL value.

**Comment by Lykdraft** [comment]
Hmm. But remember that I still can/could move the Line with the cursor. The Line itself was never stuck.

I'll try to pinpoint what I do before that happens.

**Comment by kmatheussen** [comment]
I think it's as simple as that it doesn't scroll down if you are playing. Here's the current code:
```
  if(!is_playing())
    ScrollEditorDown(window,g_downscroll);
```
(i.e. if not playing, then scroll down)

There should be an exception, that it should always scroll down if you are using the play cursor.

**Comment by kmatheussen** [comment]
fixed in 3.8.5
